src/common/playlists/PlayCard.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { BsPlayCircleFill } from 'react-icons/bs';
+import { BiLogoTypescript, BiLogoJavascript } from 'react-icons/bi';
+
+import Shimmer from 'react-shimmer-effect';
+import Like from 'common/components/Like/Like';
+import userImage from 'images/user.png';
+
+function PlayCard({ play, cover, likeObject }) {
+  return (
+    <Link to={`/plays/${encodeURI(play.github.toLowerCase())}/${play.slug}`}>
+      <div className="play-card-container">
+        <div className="play-thumb-container">
+          <Shimmer>
+            <img alt="" className="play-card-thumb-img" src={cover} />
+          </Shimmer>
+          <BsPlayCircleFill className="play-icon" color="white" size={80} />
+        </div>
+
+        {/* <div className="border" /> */}
+        <div className="card-header">{play.name}</div>
```

Comment:
need to make the `play` name to `left-align` it's a dynamic name

Reply:
it will not look good as in design we often seen mixture of centre and side aligned

Reply:
try once & send us the UI design here `before` & `after`

Reply:
<img width="322" alt="Screenshot 2024-02-21 at 21 11 46" src="https://github.com/reactplay/react-play/assets/92959398/e2e9b1c9-68b8-46ab-be92-6e71622bb8ca">
<img width="338" alt="Screenshot 2024-02-21 at 21 12 10" src="https://github.com/reactplay/react-play/assets/92959398/28ee53b1-cf12-49a0-8b6f-3396aeae0019">

Reply:
this is before and after

Reply:
done

Reply:
I think we are Good to Go

Reply:
@priyankarpal